### PR TITLE
1 technical info

### LINKED ISSUE #1: Create page with technical information about TermIt
It should contain:
- Basic technologies
- Architecture
- Info about custom libraries we use

### PR DESCRIPTION
**Pull request by ledsoft**
Resolves #1

### CODE REVIEW

File: _pages/technical-details.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+### TermIt Frontend
+
+The frontend is a client-side Web application written in TypeScript using the [React framework](https://reactjs.org/). We have used React for its simplicity of use,
+and the vast ecosystem of libraries one can utilize.
+
+The frontend source code can be found in its [GitHub repository](https://github.com/kbss-cvut/termit-ui), which also contains some basic developer documentation.
+
+### Annotace
```
Comment (psiotwo) [inline _pages/technical-details.markdown:41]: Integration with annotace

File: _pages/technical-details.markdown [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+layout: single
+#title: Technical Details
+permalink: /technical-info
+header:
+  image: /assets/images/data-model-header.jpg
+  caption: "Photo credit: [**Toby56**](https://unsplash.com/@toby56?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+---
+
+TermIt is an information system with a strong formal background. This page contains some technical background of TermIt.
```
Comment (psiotwo) [inline _pages/technical-details.markdown:13]: "TermIt is an information system with a strong formal background."

Does not say much IMO, would remove it.

File: _pages/technical-details.markdown [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+The frontend is a client-side Web application written in TypeScript using the [React framework](https://reactjs.org/). We have used React for its simplicity of use,
+and the vast ecosystem of libraries one can utilize.
+
+The frontend source code can be found in its [GitHub repository](https://github.com/kbss-cvut/termit-ui), which also contains some basic developer documentation.
+
+### Annotace
+
+One of the more interesting features of TermIt is its ability to analyze content of document in order to find occurrences of terms and suggest new terms based on their
+significance in the text. Our goal in this regard was to not create a monolith. Instead, the text analysis functionality has been developed separately to enable its
```
Comment (psiotwo) [inline _pages/technical-details.markdown:44]: TermIt is quite monolithic IMO - identifier generation service, user management, authorization, etc. could be considered separate services. The reason, why annotace was put aside was not to save TermIt from being monolith, but the licensing issues.

File: index.html [hunk 4]
```diff
@@ -20,7 +20,7 @@
     title: "Tutorial"
     excerpt: "TermIt solves various problems, from creating vocabularies to annotating resources."
     url: "/tutorial"
-    btn_label: "Learn me how"
+    btn_label: "Tell me how"
```
Comment (psiotwo) [inline index.html:23]: Show :-) ?

**Review by psiotwo — COMMENTED** [review]
see comments

File: _pages/technical-details.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+The frontend is a client-side Web application written in TypeScript using the [React framework](https://reactjs.org/). We have used React for its simplicity of use,
+and the vast ecosystem of libraries one can utilize.
+
+The frontend source code can be found in its [GitHub repository](https://github.com/kbss-cvut/termit-ui), which also contains some basic developer documentation.
+
+### Integration with Annotace
+
+One of the more interesting features of TermIt is its ability to analyze content of document in order to find occurrences of terms and suggest new terms based on their
+significance in the text. The text analysis functionality has been developed separately to enable its reuse. 
+This library is called **Annotace**, it is open source and can be found on [GitHub](https://github.com/kbss-cvut/annotace) as well.
```
Comment (MichalMed) [inline _pages/technical-details.markdown:44]: This open source library **Annotace** is located on [GitHub](https://github.com/kbss-cvut/annotace) as well.

**Review by psiotwo — COMMENTED** [review]
lgtm

File: _pages/technical-details.markdown [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## Interesting Libraries
```
Comment (MichalMed) [inline _pages/technical-details.markdown:47]: i do not like the word Interesting. It assumes other stuff is not interesting. What about "remarkable" or "notable"

File: _pages/technical-details.markdown [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+TermIt uses several custom libraries and components that may be of interest for other Semantic Web developers. Here is a short list with some basic details 
+and links where you can find more.
```
Comment (MichalMed) [inline _pages/technical-details.markdown:50]: links with more details

File: _pages/technical-details.markdown [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+#### JOPA
+
+[JOPA](https://github.com/kbss-cvut/jopa) (Java OWL Persistence API) is a persistence library for semantic data. 
+It allows working with POJO-based domain model which is stored as RDF data in a triple store. 
+Heavily inspired by JPA and its implementation, JOPA contains all the necessary bells and whistles (transactions, query result mapping, caching) to make development
+of a Semantic Web-based information system as smooth as possible.
+
+There is also a [library](https://github.com/ledsoft/jopa-spring-transaction) for integrating JOPA with the declarative 
+transaction management of Spring (think `@Transactional`) that we use in TermIt.
```
Comment (MichalMed) [inline _pages/technical-details.markdown:60]: used in TermIt

File: index.html [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: splash
 author_profile: false
-excerpt: "TermIt is a tool to manage vocabularies, terms and resources in which are the terms defined and used. It is focused on managment of terms based on their semantics -- i.e. two 'same' terms in different vocabularies may have different menaing."
+excerpt: "TermIt is a tool to manage vocabularies, terms and resources in which are the terms defined and used. It is focused on management of terms based on their semantics -- i.e. two 'same' terms in different vocabularies may have different menaing."
```
Comment (MichalMed) [inline index.html:4]: menaing -> meaning

File: index.html [hunk 10]
```diff
@@ -20,7 +20,7 @@
     title: "Tutorial"
     excerpt: "TermIt solves various problems, from creating vocabularies to annotating resources."
     url: "/tutorial"
-    btn_label: "Learn me how"
+    btn_label: "Show me how"
```
Comment (MichalMed) [inline index.html:23]: I wanted to use different verb than in deployments, but as far as we're cancelling deployments page, i think its OK

File: index.html (hunk 4 above)
Comment (MichalMed) [reply index.html]: definitely tell, show is in the second article

File: _pages/technical-details.markdown [hunk 11]
```diff
@@ -4,35 +4,36 @@
...
-permalink: /technical-info
+permalink: /technical-details
 header:
   image: /assets/images/data-model-header.jpg
   caption: "Photo credit: [**Toby56**](https://unsplash.com/@toby56?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+toc: true
 ---
 
-This page contains some technical background of TermIt. It describes the architecture of the system, interesting technologies it uses, and the data model at its core.
+This page contains some technical background of TermIt. It describes the architecture of the system, libraries developed by us that it uses, and the data model at its core.
```
Comment (MichalMed) [inline _pages/technical-details.markdown:14]: custom developed libraries

File: _pages/technical-details.markdown [hunk 12]
```diff
@@ -44,10 +45,10 @@ significance in the text. The text analysis functionality has been developed sep
 This library is called **Annotace**, it is open source and can be found on [GitHub](https://github.com/kbss-cvut/annotace) as well.
 
 
-## Interesting Libraries
+## Notable Libraries
```
Comment (MichalMed) [inline _pages/technical-details.markdown:48]: Custom :)

**Review by MichalMed — COMMENTED** [review]
:)

**Pull request merged**